### PR TITLE
docs: add discord intergration document

### PR DESCRIPTION
**Pull request by Michaelg22**
### Related Issue

Solving Issue: #xxx

### PR Type

- [ ] Fix typos or format(punctuation, space, indentation, code block, etc.)
- [ ] Update inappropriate or misunderstanding content
- [ ] Add missing content or new documents
- [ ] Delete useless content or outdated documents
- [ ] Localize or translate documents
- [ ] Deal with GitHub action files and scripts
- [ ] Deal with chores

### CODE REVIEW

File: content/cloud-docs/source/zoho-crm/setup.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
---- 
+q---
```
Comment (Michaelg22) [inline content/cloud-docs/source/zoho-crm/setup.md:1]:
```suggestion
--- 
```

File: content/vanus-ai/how-to/discord.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Discord Integration
+This documentation will teach you how to integrate an AI into Discord.
+Integration is a paid user feature. Therefore, only users with subscriptions or 14 days trials can use this feature.
+
+## how to Setup a Discord Integration
+
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:6]: Replace *how* with *How*

File: content/vanus-ai/how-to/discord.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+### Step 1: Create an AI Application
+1. Go to [ai.vanus.ai](https://ai.vanus.ai) and sign in.
+2. Create an new AI app by clicking **Create AI**.
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:9]: Create *a* new AI app by clicking *Create App*

File: content/vanus-ai/how-to/discord.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+![](images/discord-createapp.png)
+3. Enter a name, the model, upload to the knowledge base, and click **Create**.
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:11]: Enter a name, the *app* model, upload to the knowledge base, and click **Create**.

File: content/vanus-ai/how-to/discord.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+![](images/discord-enterNameModelKnownledge.png)
+4. Go to the **integration** TAB and click on the discord integration.
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:13]: Go to the **Integration** tab and click on the discord integration.

File: content/vanus-ai/how-to/discord.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+![](images/discord-integration.png)
+
+### Step 2: Create a Discord Channel
+1. Go to the discord App or website at [discord](https://discord.com)
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:17]: Go to the Discord App or Website at [discord](https://discord.com)

File: content/vanus-ai/how-to/discord.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+1. Visit [Discord Developer Portal](https://discord.com/developers/applications)
+2. Click **New Application**.
+![](images/discord-newapplication.png)
+3. Give a name to your bot, accept the terms and conditions and click **Create**.
+![](images/discord-givebotname.png)
+4. From the side menu go to **bot**, and in Privileged Gateway Intents enable **MESSAGE CONTENT INTENT**.
+![](images/discord-sidemenuclickbot.png)
+5. Click **Reset token** and **Copy**.
+![](images/discord-resettoken.png)
+6. Go back to Vanus AI paste the Token, give the integration a name and press Create.
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:32]: Go back to Vanus AI, paste the token, give the integration a name, and press Create.

**Review by ehisakhile — CHANGES_REQUESTED** [review]
Single line comments to be reviewed

File: content/vanus-ai/how-to/discord.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
+# Discord Integration
+This documentation will teach you how to integrate an AI into Discord.
+Integration is a paid user feature. Therefore, only users with subscriptions or 14 days trials can use this feature.
+
+## how to Setup a Discord Integration
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:5]:
```suggestion
## How to Setup a Discord Integration
```

File: content/vanus-ai/how-to/discord.md (hunk 3 above)
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:9]:
```suggestion
2. Create a new AI App by clicking **Create App** button.
```

File: content/vanus-ai/how-to/discord.md (hunk 4 above)
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:11]:
```suggestion
3. Enter a Name, the App Model, upload to the Knowledge Base, and click **Create**.
```

File: content/vanus-ai/how-to/discord.md (hunk 5 above)
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:13]:
```suggestion
4. Go to the **Integration** tab and click on the Discord integration.
```

File: content/vanus-ai/how-to/discord.md (hunk 6 above)
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:17]:
```suggestion
1. Go to the Discord App or Website at [Discord](https://discord.com)
```

File: content/vanus-ai/how-to/discord.md (hunk 7 above)
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:32]:
```suggestion
6. Go back to Vanus AI, paste the token, give the integration a name, and press **Create**.
```

File: content/vanus-ai/how-to/discord.md [hunk 9]
```diff
@@ -0,0 +1,54 @@
...
+6. Go back to Vanus AI paste the Token, give the integration a name and press Create.
+![](images/discord-gobacktovanusai.png)
+7. Lastly, to add an AI bot to a channel, go back to the application inside the developer portal.
+8. Under the sidemenu 0Auth2 and URL Generator.
+![](images/discord-sidemenu0auth2.png)
+9. Select bot under scopes.
+
+10. Under Bot Permission (ReadMessages/ViewChannel, Send Messages, Mention Everyone, Use External Emojis, Send TTS Messages, Send Messages In threads,Create public, threads Create private threads) or Administrator.
+![](images/discord-underbotpermission.png)
+11. Copy the URL and visit it from another tab..
```
Comment (ehisakhile) [inline content/vanus-ai/how-to/discord.md:41]:
```suggestion
11. Copy the URL and open it in a new tab.
```

**Review by ehisakhile — CHANGES_REQUESTED** [review]
Updated Changes

**Pull request merged**